smoothie-lifecycle-ktp/build.gradle

```diff
@@ -11,8 +11,8 @@ android {
   compileSdkVersion 28
 
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_1_7
+    targetCompatibility JavaVersion.VERSION_1_7
```

Comment:
why?

Reply:
To make `smoothie-lifecycle` compatible with projects that use
```
sourceCompatibility JavaVersion.VERSION_1_7
targetCompatibility JavaVersion.VERSION_1_7
```
And because all other modules use 1.7 too